frontend/src/pages/Privacy/Privacy.tsx

```diff
@@ -34,10 +29,11 @@ const Privacy = () => (
 
     <h3>Cookies</h3>
     <p>Cookies are files with a small amount of data that are commonly used as anonymous unique identifiers. These are sent to your browser from the websites that you visit and are stored on your device's internal memory.</p>
-    <p>Cookies are used by Firebase Authentication to keep you signed in on the Service.</p>
+    <p>Cookies are used to store your project data on your machine only. Clearing your cookies will remove your projects from Automatarium.</p>
```

Comment:
iirc projects are stored in localStorage, not cookies

Reply:
Ahh I thought they were the same thing for the longest time. This makes sense.